lib/jobs/dell-racadm-set-bios-job.js

```diff
@@ -0,0 +1,111 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+
+module.exports = racadmSetBIOSJobFactory;
+di.annotate(racadmSetBIOSJobFactory, new di.Provide('Job.DellRacadm.SetBIOS'));
+di.annotate(racadmSetBIOSJobFactory, new di.Inject(
+    'Job.Base',
+    'JobUtils.RacadmTool',
+    'Logger',
+    'Util',
+    'Assert',
+    'Promise',
+    '_',
+    'Services.Encryption',
+    'Services.Lookup',
+    'Constants',
+    'Services.Waterline'
+));
+
+function racadmSetBIOSJobFactory(
+    BaseJob,
+    racadmTool,
+    Logger,
+    util,
+    assert,
+    Promise,
+    _,
+    encryption,
+    lookup,
+    Constants,
+    waterline
+) {
+    var logger = Logger.initialize(racadmSetBIOSJobFactory);
+    /**
+     *
+     * @param {Object} options
+     * @param {Object} context
+     * @param {String} taskId
+     * @constructor
+     */
+    function RacadmSetBiosJob(options, context, taskId) {
+        RacadmSetBiosJob.super_.call(this, logger, options, context, taskId);
+
+        this.nodeId = this.context.target;
+        this.userName= options.username;
+        this.passWord = options.password;
```

Comment:
It's better to use `password` not `passWord`, because for security reason, the password will be redact so that the plain password will not be printed in log file. But `passWord` will not be redact by design.